mesonbuild/environment.py

```diff
@@ -353,6 +353,16 @@ def detect_cpu_family(compilers: CompilersDict) -> str:
         # AIX always returns powerpc, check here for 64-bit
         if any_compiler_has_define(compilers, '__64BIT__'):
             trial = 'ppc64'
+    # While trying to read machine infomations on iOS/watchOS/tvOS, it returns
+    # the machine model name rather than architecture, e.g. `uname -m` would get
+    # 'iPhone13,1' but not 'arm64' on iPhone 12 mini. Same thing happens on other
+    # Apple mobile devices, but `uname -p` is having correct values, so use
+    # platform.processor instead of the default platform.machine
+    elif any(appleDevices in trial for appleDevice in ['iphone', 'ipad', 'ipod', 'watch', 'appletv', 'audioaccessory']):
```

Comment:
What is appleDevices? Not the variable you defined later on the line...

Reply:
sorry for my mistake, just corrected these typos